src/nodes/NodeConnectionManager.ts

```diff
@@ -129,6 +128,22 @@ class NodeConnectionManager {
    * Default timeout for RPC handlers
    */
   public readonly rpcCallTimeoutTime: number;
+  /**
+   * Used to track active hole punching attempts
+   */
+  protected activePunchMap = new Map<string, PromiseCancellable<void>>();
```

Comment:
This should be `activePunchPMap`. Do we use `Map` and `Set` suffixes commonly across the codebase? I don't remember it.

Reply:
No, usually we don't include them.

I'll rename this to `activePunchPs`?

Reply:
I've updated the names and commentary.